docs-conceptual/azps-6.1.0/install-az-ps.md

```diff
@@ -49,6 +50,18 @@ platforms. Run the following command from a PowerShell session:
 ```powershell-interactive
 Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force
 ```
+Then you need to import the module:
+
+```powershell-interactive
+Import-Module az.accounts
+```
+In case you ran into the error `Import-Module : cannot be loaded because running scripts is disabled on this system.` ,
+you need to open a new PowerShell instance using `Run as Administrator` option, then run the following command:
+
+```powershell-interactive
+
+Set-ExecutionPolicy RemoteSigned
+```
```

Comment:
Yes. This can be an issue. You can set this option for the current user to avoid having to run PowerShell elevated as an admin.

```powershell
Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
```